config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -80,11 +66,53 @@ periodics:
             value: "us-west-2"
         securityContext:
           privileged: true
+        resources:
+          requests:
+            cpu: 1
+            memory: "4Gi"
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
     testgrid-tab-name: periodic-e2e-release-0-4
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
     testgrid-num-failures-to-alert: "2"
+- name: periodic-cluster-api-provider-aws-e2e-conformance-master
```

Comment:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1501 forward ports the e2e conformance testing from the `release-0.4` branch, so this puts in place the periodic job to start exercising it.